0000-payment-pointers/0000-payment-pointers.md

```diff
@@ -0,0 +1,126 @@
+
+The client should then initiate an https connection with the endpoint server and perform an http GET request to the endpoint URL.
+
+The cleint should express the protocols it supports by including the media types of the protocol messages in the `Accept` header of the request.
+
+# IANA Considerations
+
+## Well-Known URI
+
+This specification registers a new well-known URI. 
+
+URI suffix:  "pay"
+
+Change controller:  Interledger Payments Community Group (W3C)
+
+Specification document(s):  This document
+
+# Security Considerations
+
+Payment pointers have many of the same security risks as URLs so it follows that the security considerations are the same even though payment pointers do not in and of themselves pose a security threat.  
+
+However, as payment pointers are used to provide a compact set of instructions for initiation of a payment, care must be taken to properly interpret the data within a payment pointer, to prevent payments being made to the worng recipient or leaking of sensitive data.
+
+## Reliability and Consistency
+
+There is no guarantee that once a payment pointer has been used resolve a payment setup protocol receiver endpoint, the same service will be hosted at that URI in the future.
+
+## Semantic Attacks and Phishing
+
+Because payment pointers only support a limited profile of the data in a URL, not all of the attacks described in RFC3986 apply to payment pointers.
+
+However it is possible for a malicious actor to construct a payment pointer that appears to point to a trusted receiver but in fact poimnts to a malicious actor. This is especially so when using internationalized domain names in the payment pointer as these can be constcuted to appear as if there is a path separator (`/`) but this may in fact be a non-latin character in the _host_ portion of the pointer.
+
+As a result the pointer is constructed that is intended to mislead a human user by appearing to identify one (trusted) naming authority while actually identifying a different authority hidden behind the noise. 
+
+As detailed in [RFC7230], the "https" scheme (Section 2.7.2) is intended to prevent (or at least reveal) many of these potential attacks on establishing the authority behind a pointer, provided that the negotiated TLS connection is secured and the client properly verifies that the communicating server's identity matches the target URI's authority component (see [RFC2818]).Correctly implementing such verification can be difficult (see [Georgiev]).
+
+Payment Setup Protocols should take heed of this risk in their design and provide a mechainsim for the sender to reliably verify the receiver's identity. This may be as simple as relying on the certificate of the receving endpoint server to identify the receiver, or may be handled explicitly within the protocol.
```

Comment:
This seems to downplay the risks. If you use SPSP with the wrong endpoint, you're sending money to the wrong person. 

The client should check that the cert matches the endpoint you query, but that doesn't help you if you've just queried the wrong endpoint. Furthermore, if the payment pointer resolves to a URL with a redirect, the endpoint you're talking to will have a cert that has no relationship to the payment pointer at all.